sdks/RepoToolset/tools/Sign.proj

```diff
@@ -19,6 +25,8 @@
     </Exec>
 
     <ItemGroup>
+      <SignToolArgs Condition="'$(OrchestrationMode)' == 'true'" Include='-orchestrationmode' />
```

Comment:
>"'$(OrchestrationMode)' == 'true'" Include='-orchestrationmode' /> [](start = 30, length = 66)

Why do we need extra parameter? Isn't the presence of ```-orchestrationconfigfile``` sufficient?

Reply:
There're two modes:

- orchestrationConfigFile produces files like the one shown in step 1 above.
- OrchestrationMode is how I decide what your JSON will look like for executing Step 2;  int hat case the normal config file is expected to already contain hashes of each file.